client/store/auth.ts

```diff
@@ -33,6 +35,12 @@ export const auth: Auth = {
     state.domain = null;
     state.email = null;
     state.isAdmin = false;
+    const auth2 = window.gapi.auth2.getAuthInstance();
+    if (auth2 != null) {
+      auth2.signOut().then(
+           auth2.disconnect().then(console.log('LOGOUT SUCCESSFUL'))
+       )
+    }
```

Comment:
I don't know exactly why, but this is throwing an error in the client when I try to logout with an account created with e-mail and password (instead of using Google OAuth). I guess it needs to check if the user has logged in with Google and if this option is available (using `DISALLOW_GOOGLE`).